courtscraper/spiders/court_calls.py

```diff
@@ -0,0 +1,222 @@
+from abc import ABC
+from datetime import datetime, timedelta
+
+from scrapy import Spider, Request
+from scrapy.http import FormRequest
+from scrapy.exceptions import CloseSpider
+from scrapy.spidermiddlewares.httperror import HttpError
+
+from lxml import html
+
+from scripts.hash import dict_hash
+
+
+class CourtCallSpider(ABC, Spider):
```

Comment:
Why is this an ABC, when we never subclass it?

Reply:
Good point! I'll remove it.